### PR TITLE
 Add a sample for listing groups with external access.

### PR DESCRIPTION
**Pull request by erickoledadevrel**

### CODE REVIEW

File: admin_sdk/groups_settings/README.md [hunk 1]
```diff
@@ -1,18 +1,37 @@
...
-## Run
+### Run
 
 ```
 python quickstart.py
 ```
+
+## Detect External Access
+
+This script lists all groups in your domain with some form of external acess
```
Comment (grant) [inline admin_sdk/groups_settings/README.md]: acess

**Comment by grant** [comment]
Lint if you want, but the lint errors for the whole repo will be handled later.
```py
pip install pylint
pylint admin_sdk/groups_settings/detect_external_access.py
```

File: admin_sdk/groups_settings/detect_external_access.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    Given the group email and its settings, checks some of its settings and
+    prints them if the group has external access.
+    """
+    who_can_view_group = settings['whoCanViewGroup']
+    who_can_join = settings['whoCanJoin']
+    allow_external_members = settings['allowExternalMembers']
+    who_can_post_message = settings['whoCanPostMessage']
+    who_can_view_membership = settings['whoCanViewMembership']
+    if (who_can_view_group == ANYONE_CAN_VIEW_GROUP
+            or who_can_join == ANYONE_CAN_JOIN_GROUP
```
Comment (grant) [inline admin_sdk/groups_settings/detect_external_access.py]: PEP8: line break before binary operator
In this case, the `or` should go after the cnf.

http://pep8online.com/checkresult

File: admin_sdk/groups_settings/detect_external_access.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+EXTERNAL_MEMBERS_CAN_JOIN = 'true'
+
+# External access for who can post messages
+ANYONE_CAN_POST_MESSAGE = 'ANYONE_CAN_POST'
+
+# External access for who can view members
+ANYONE_CAN_VIEW_MEMBERSHIP = 'ANYONE_CAN_VIEW'
+
+def get_credentials():
+    """Gets valid user credentials from storage.
```
Comment (grant) [inline admin_sdk/groups_settings/detect_external_access.py]: `"""` should be on it's own line if we're being consistent.

File: admin_sdk/groups_settings/detect_external_access.py (hunk 2 above)
Comment (erickoledadevrel) [reply admin_sdk/groups_settings/detect_external_access.py]: Done.

File: admin_sdk/groups_settings/detect_external_access.py (hunk 3 above)
Comment (erickoledadevrel) [reply admin_sdk/groups_settings/detect_external_access.py]: Done

**Pull request merged**